python/servo/command_base.py

```diff
@@ -411,7 +411,7 @@ def android_support_dir(self):
     def android_build_dir(self, dev):
         return path.join(self.get_target_dir(), "arm-linux-androideabi", "debug" if dev else "release")
 
-    def ensure_bootstrapped(self, targets=[]):
+    def ensure_bootstrapped(self, target=""):
```

Comment:
The default here should be `target=None`, not `target=""`. We only need to do the target-related things if we are passed a target explicitly.